### PR TITLE
Material card UI

### PR DESCRIPTION
**Pull request by robin-papp**
Updated the user card UI structure on the Partners and Find Partners pages to match that of the Request page. Did a few tweaks to UI like labels to buttons

### CODE REVIEW

**Review by mohsinqadeer — APPROVED** [review]

**Review by mohsinqadeer — APPROVED** [review]

**Review by mohsinqadeer — APPROVED** [review]

**Review by mohsinqadeer — APPROVED** [review]

**Review by mohsinqadeer — APPROVED** [review]

**Review by mohsinqadeer — APPROVED** [review]

**Review by mohsinqadeer — APPROVED** [review]

**Review by mohsinqadeer — APPROVED** [review]

**Review by mohsinqadeer — APPROVED** [review]

**Review by mohsinqadeer — APPROVED** [review]

**Review by mohsinqadeer — APPROVED** [review]

**Review by EAkkanen — APPROVED** [review]
Looks alright

**Comment by ayojava** [comment]
its okay

File: src/containers/BrowseMatch/index.js [hunk 1]
```diff
@@ -170,72 +179,72 @@ class BrowseMatch extends React.Component
...
-                        <Typography className={classes.heading}>
-                            Possible matches who can teach you {item.languageName} - <strong>{item.matches.length} match(es) &nbsp;&nbsp;&nbsp;&nbsp;</strong>
+            <div key={item.languageName}>
+                <ExpansionPanel className={classes.expansionPan} defaultExpanded={this.state.isDefaultExpand}>
+                    <ExpansionPanelSummary
+                        expandIcon={<ExpandMoreIcon />}
+                        aria-controls="panel1a-content"
+                        id="panel1a-header"
+                    >
+                        <Typography variant="h6" className={classes.headingxxxxx}>
```
Comment (janakavishwajith) [inline src/containers/BrowseMatch/index.js]: this class name? headingxxxxx

File: src/containers/BrowseMatch/index.js [hunk 2]
```diff
@@ -170,72 +179,72 @@ class BrowseMatch extends React.Component
...
+                            Possible matches who can teach you {item.languageName} <strong> - {item.matches.length} match(es) &nbsp;&nbsp;&nbsp;&nbsp;</strong>
```
Comment (janakavishwajith) [inline src/containers/BrowseMatch/index.js:178]: Isn't it better to make the adjustment with the css, rather than adding &nbsp;

**Review by janakavishwajith — CHANGES_REQUESTED** [review]
Some minor changes to be done...

File: src/containers/BrowseMatch/index.js (hunk 1 above)
Comment (robin-papp) [reply src/containers/BrowseMatch/index.js]: The heading class does no visual changes. The code is full of unnecessary style declarations. I will go through them and delete these classes

File: src/containers/BrowseMatch/index.js (hunk 2 above)
Comment (robin-papp) [reply src/containers/BrowseMatch/index.js:178]: You mean the variant? That's standard Material-UI stuff. I'm just making it match the Request page

**Review by janakavishwajith — APPROVED** [review]

**Pull request merged**